deploy/scripts/src/authentication/test.ts

```diff
@@ -349,34 +317,46 @@ export function signUp(): void {
     sleep(1)
 
     // B01_SignUp_12_Logout
-    timeGroup(groups[17], () => {
-      // 01_RPStub
-      res = timeGroup(groups[18].split('::')[1], () => http.get(env.rpStub + '/logout', { redirects: 0 }), {
-        isStatusCode302
-      })
-      // 02_OIDCCall
-      res = timeGroup(groups[19].split('::')[1], () => http.get(res.headers.Location, { redirects: 0 }), {
-        isStatusCode302
-      })
-      // 03_AuthCall
-      res = timeGroup(groups[20].split('::')[1], () => http.get(res.headers.Location), {
-        isStatusCode200,
-        ...pageContentCheck('You have signed out')
-      })
-    })
+    logout(groupMap.signUp, 17, 18, 19, 20)
   }
 
   iterationsCompleted.add(1)
 }
 
-export function signIn(): void {
+export function logout(
+  groupMap: readonly string[],
+  groupIndex1: number,
+  groupIndex2: number,
+  groupIndex3: number,
+  groupIndex4: number
+): void {
   let res: Response
-  const groups = groupMap.signIn
-  const userData = dataSignIn[execution.scenario.iterationInInstance % dataSignIn.length]
-  iterationsStarted.add(1)
+  const groups = groupMap
+
+  // B02_SignIn_09_Logout
+  timeGroup(groups[groupIndex1], () => {
+    // 01_RPStub
+    res = timeGroup(groups[groupIndex2].split('::')[1], () => http.get(env.rpStub + '/logout', { redirects: 0 }), {
```

Comment:
```suggestion
    logout(groups.slice(17,20))
  }

  iterationsCompleted.add(1)
}

export function logout(
  groups: readonly string[],
): void {
  let res: Response

  // Logout
  timeGroup(groups[0], () => {
    // 01_RPStub
    res = timeGroup(groups[1].split('::')[1], () => http.get(env.rpStub + '/logout', { redirects: 0 }), {
```
Suggest to:
- Use the already defined `groups` variable in place of `groupMap.SignUp` here
- Pass in a slice of the group names to avoid having to use four index paramaters
- Update the comments which relate to specific scenarios and step numbers